src/command.rs

```diff
@@ -18,7 +18,9 @@ COMMANDS:
         Task ID, language, and the language option may be automatically deduced
         by the server, in which case they do not need to be supplied as options.
     submissions [-c] (-t)       List previous submissions to a task.
-    submission [-c] <id>        Show details about the submission with given ID.
+    submission [-c] (<id> | (-t) [<n>])
+        Show details about the submission with given ID or specify a task and 
+        your nth latest submission to it (default 1).
```

Comment:
Here the term `nth latest` is used but in some variables `NthLast` is used. I don't know which one is correct (maybe both?) but I feel like "nth last" or "nth to last" (where the "to" could be omitted in code) sounds better.